classes/ActionScheduler_wpPostStore.php

```diff
@@ -265,44 +317,87 @@ public function query_actions( $query = array() ) {
 			$sql_params[] = $query['claimed'];
 		}
 
-		switch ( $query['orderby'] ) {
-			case 'hook':
-				$orderby = 'p.title';
-				break;
-			case 'group':
-				$orderby = 't.name';
-				break;
-			case 'modified':
-				$orderby = 'p.post_modified';
-				break;
-			case 'date':
-			default:
-				$orderby = 'p.post_date_gmt';
-				break;
-		}
-		if ( strtoupper($query['order']) == 'ASC' ) {
-			$order = 'ASC';
-		} else {
-			$order = 'DESC';
-		}
-		$sql .= " ORDER BY $orderby $order";
-		if ( $query['per_page'] > 0 ) {
-			$sql .= " LIMIT %d, %d";
-			$sql_params[] = $query['offset'];
-			$sql_params[] = $query['per_page'];
+		if ( 'select' === $select_or_count ) {
+			switch ( $query['orderby'] ) {
+				case 'hook':
+					$orderby = 'p.title';
+					break;
+				case 'group':
+					$orderby = 't.name';
+					break;
+				case 'modified':
+					$orderby = 'p.post_modified';
+					break;
+				case 'date':
+				default:
+					$orderby = 'p.post_date_gmt';
+					break;
+			}
+			if ( strtoupper($query['order']) == 'ASC' ) {
```

Comment:
Minor formatting adjustment needed within the `strtoupper()` call. Also, Yoda condition and `===` should be used here.